src/setupTests.js

```diff
@@ -0,0 +1,4 @@
+import Enzyme from 'enzyme';
```

Comment:
you don't need this file - this is for React app tests - the starter code was confusing :/